nuxeo-compound-documents/src/test/java/org/nuxeo/compound/documents/CompoundDocumentsFeature.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * (C) Copyright 2022 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.nuxeo.compound.documents;
+
+import org.nuxeo.ecm.restapi.test.RestServerFeature;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.RunnerFeature;
+
+/** @since 2021.0 */
+@Features(RestServerFeature.class)
+@Deploy("org.nuxeo.ecm.platform.filemanager")
+@Deploy("org.nuxeo.ecm.platform.video:OSGI-INF/core-types-contrib.xml")
+@Deploy("org.nuxeo.ecm.platform.audio.core:OSGI-INF/core-types-contrib.xml")
+@Deploy("org.nuxeo.ecm.platform.picture.core:OSGI-INF/picture-schemas-contrib.xml")
```

Comment:
I do need each one of them because of the facets and assertions. https://github.com/nuxeo/nuxeo-compound-documents/blob/2021/nuxeo-compound-documents/src/test/java/org/nuxeo/compound/documents/TestCompoundDocumentRest.java#L109-L110

Unless we judge since subtypes are not enforcers, we can get rid of that part of the tests ?

Reply:
👍 Oh, right... let's discuss it later on with Manon and Matt to see what to do with subtypes.